### PR TITLE
feat: upgrade to Ansible 7

### PR DESCRIPTION
**Pull request by paulswartz**

### CODE REVIEW

File: linux/roles/crowdstrike/tasks/main.yml [hunk 1]
```diff
@@ -1,24 +1,26 @@
 ---
 - name: Download Falcon sensor
-  register: download_falcon_sensor
-  aws_s3:
+  register: crowdstrike_download
+  amazon.aws.s3_object:
     region: "{{ s3_config_region }}"
     bucket: "{{ s3_config_bucket }}"
-    object: "{{ s3_config_prefix | trim }}/{{ crowdstrike_path}}"
```
Comment (krisrjohnson21) [inline linux/roles/crowdstrike/tasks/main.yml:7]: Question: Does removing this `trim` cause any downstream issues?

Comment (paulswartz) [reply linux/roles/crowdstrike/tasks/main.yml:7]: When the configuration for this value moved into the separate file (https://github.com/mbta/on_prem_deploy/pull/19/commits/eb8d96ba9900f1ba4e4b942d6213381af5100d34) it lost the trailing newline that it had before.

File: linux/roles/ecs_anywhere/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+
+ecs_anywhere_aws_gpg_key: BCE9D9A42D51784F
+ecs_anywhere_install_prefix: "https://amazon-ecs-agent.s3.amazonaws.com"
+ecs_anywhere_region: us-east-2
```
Comment (krisrjohnson21) [inline linux/roles/ecs_anywhere/defaults/main.yml:5]: Question: Should this be `us-east-1`?

Comment (paulswartz) [reply linux/roles/ecs_anywhere/defaults/main.yml:5]: The defaults are for the test account, which has all the resources in us-east-2. The staging/production values are overridden to us-east-1: https://github.com/mbta/on_prem_deploy/pull/19/files#diff-e77357a3abfee2bac980327808361db299a7a0585b68b184d6fe0a132bd0c87eR19-R35

File: linux/roles/ecs_anywhere/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ecs_anywhere_cluster: linux-test
+# these are encrypted, stored in group_vars/<group>/vault.yml
+ecs_anywhere_activation_id: "{{ vault_ecs_anywhere_activation_id }}"
+ecs_anywhere_activation_code: "{{ vault_ecs_anywhere_activation_code }}"
+
+s3_config_region: us-east-1
```
[the remark this answers is not in the document]
Comment (paulswartz) [reply linux/roles/ecs_anywhere/defaults/main.yml:11]: Yes, we're good.

**Review by krisrjohnson21 — APPROVED** [review]
Thanks for bearing with my questions! Saw the answers flow in, LGTM

**Pull request merged**